frontend/package.json

```diff
@@ -43,7 +43,6 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-vue": "^9.3.0",
     "happy-dom": "^7.6.6",
-    "npm-run-all": "^4.1.5",
```

Comment:
Looks like `build` uses **npm-run-all** (`run-p`) so removing this breaks the TS building part

Reply:
Thanks for the catch. I'll take another look and see if we can rewrite the build or revert this change.